pyproject.toml

```diff
@@ -0,0 +1,2 @@
+[build-system]
+requires = ["cython"]
```

Comment:
Do you think we need to add setuptools and wheel to this list?

See, https://www.python.org/dev/peps/pep-0518/#build-system-table